src/actions/visibility.js

```diff
@@ -0,0 +1,41 @@
+export function showLoginDialog () {
```

Comment:
The code is there but effectively not executed since oVirt SSO overrides the flow.
In the future, even with remote deployment, the oVirt SSO will be still used.

The LoginForm and related code can be removed.
But to avoid bugs, I would postpone it to a followup patch.

Let's keep this PR effort as much simple as possible.